test/framework/clusterctl/logger/log_file.go

```diff
@@ -36,7 +36,7 @@ func CreateLogFile(input CreateLogFileInput) *LogFile {
 	filePath := filepath.Join(input.LogFolder, input.Name)
 	Expect(os.MkdirAll(filepath.Dir(filePath), 0750)).To(Succeed(), "Failed to create log folder %s", filepath.Dir(filePath))
 
-	f, err := os.Create(filePath) //nolint:gosec // No security issue: filepath is safe.
+	f, err := os.OpenFile(filePath, os.O_RDWR|os.O_CREATE|os.O_APPEND, 0666) //nolint:gosec // No security issue: filepath is safe.
```

Comment:
We should probably change the func name as well. WDYT about `OpenLogFile`?

Reply:
Should we create a new func then too?